fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/socialmedia/SocialMediaSendFragment.java

```diff
@@ -47,32 +49,17 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     super.onViewCreated(view, savedInstanceState);
 
     setupSendChirpButton();
-
-    // Subscribe to "send chirp" event
-    mSocialMediaViewModel
-        .getSendNewChirpEvent()
-        .observe(
-            getViewLifecycleOwner(),
-            booleanEvent -> {
-              Boolean event = booleanEvent.getContentIfNotHandled();
-              if (event != null) {
-                sendNewChirp();
-              }
-            });
   }
 
   private void setupSendChirpButton() {
-    mSocialMediaSendFragBinding.sendChirpButton.setOnClickListener(
-        v -> mSocialMediaViewModel.sendNewChirpEvent());
+    mSocialMediaSendFragBinding.sendChirpButton.setOnClickListener(v -> sendNewChirp());
   }
 
   private void sendNewChirp() {
-    // Trying to send a chirp when no LAO has been chosen in the application will not send it and
-    // make a toast appear
+    // Trying to send a chirp when no LAO has been chosen in the application will not send it, will
+    // make a toast appear and will log the error
```

Comment:
```suggestion
    // Trying to send a chirp when no LAO has been chosen in the application will not send it, it will
    // make a toast appear and it will log the error
```